### DISCUSSION TITLE
User Access Restrictions: Next Up

### DISCUSSION THREAD

**Discussion by serratoseg79** [jellyfin/jellyfin#5421] [Q&A]
Hello all!

I've been running Jellyfin for a while now, slowly adding to my library but I want to restrict some access for specific users for kids use.  I see there is a way to block access with Tags and Ratings via parental controls but the problem is that the "Next Up" and "Latest" sections on the home screen still show some of the content I'm trying to restrict, giving the restricted user full access if those videos show up in those sections.  

I basically only have two directories, Movies and TV Shows.  If I restrict access to the TV shows directory, then the Next Up and Latest sections don't appear anyway so it's not an issue then, but then they don't have any access to any tv shows.  

Has anyone got a solution to this?  Either by disabling Next Up and Latest for specific users, or an actual fix to the issue by removing them from those lists if they are blocked?

**Comment by gotton** [comment]
What if you use the label option. Settings > users > parental control > block element with this label

And you add the label to the movie or the tv show that you dont want your kid to see ?

It's not a perfect solution but it can block all element containing this label.

It would be more usefull if you could allow only the files containing the label but you only can block at this time.

**Reply by serratoseg79** [reply]
I think you mean Tags?  I don't see a label option in Parental Control.  I currently have rating blocking, tags, and access schedule in parental controls.  The Tags and rating did not work.  It would remove them from the directory but would still appear in the next up and latest sections on the home screen.

**Comment by MagicLoustic** [comment]
+ 1 for allow with tag and not the inverse !

**Reply by MagicLoustic** [reply]
thanks
